client/src/app/gui/payload_type.rs

```diff
@@ -0,0 +1,227 @@
+                media_type: Audio,
+            },
+            16 => PayloadType {
+                id: 16,
+                name: "DVI4".to_string(),
+                clock_rate_in_hz: Some(11025),
+                media_type: Audio,
+            },
+            17 => PayloadType {
+                id: 17,
+                name: "DVI4".to_string(),
+                clock_rate_in_hz: Some(22050),
+                media_type: Audio,
+            },
+            18 => PayloadType {
+                id: 18,
+                name: "G729".to_string(),
+                clock_rate_in_hz: Some(8000),
+                media_type: Audio,
+            },
+            19 => PayloadType {
+                id: 19,
+                name: "reserved (previously CN)".to_string(),
+                clock_rate_in_hz: None,
+                media_type: Audio,
+            },
+            25 => PayloadType {
+                id: 25,
+                name: "CELLB".to_string(),
+                clock_rate_in_hz: Some(90000),
+                media_type: Video,
+            },
+            26 => PayloadType {
+                id: 26,
+                name: "JPEG".to_string(),
+                clock_rate_in_hz: Some(90000),
+                media_type: Video,
+            },
+            28 => PayloadType {
+                id: 28,
+                name: "nv".to_string(),
+                clock_rate_in_hz: Some(90000),
+                media_type: Video,
+            },
+            31 => PayloadType {
+                id: 31,
+                name: "H261".to_string(),
+                clock_rate_in_hz: Some(90000),
+                media_type: Video,
+            },
+            32 => PayloadType {
+                id: 32,
+                name: "MPV".to_string(),
+                clock_rate_in_hz: Some(90000),
+                media_type: Video,
+            },
+            33 => PayloadType {
+                id: 33,
+                name: "MP2T".to_string(),
+                clock_rate_in_hz: Some(90000),
+                media_type: AudioOrVideo,
+            },
+            34 => PayloadType {
+                id: 34,
+                name: "H263".to_string(),
+                clock_rate_in_hz: Some(90000),
+                media_type: Video,
+            },
+            72..=76 => PayloadType {
+                id: 72,
+                name: "reserved".to_string(),
+                clock_rate_in_hz: None,
+                media_type: AudioOrVideo,
+            },
+            77..=95 => PayloadType {
+                id: 77,
+                name: "unassigned".to_string(),
+                clock_rate_in_hz: None,
+                media_type: AudioOrVideo,
+            },
+            _ => PayloadType {
+                id: payload_type,
+                name: "unassigned".to_string(),
+                clock_rate_in_hz: None,
+                media_type: AudioOrVideo,
+            },
+        }
+    }
+}
+
+impl std::fmt::Display for PayloadType {
```

Comment:
I don't really like the format, maybe something like:
```
Payload type: 8
Name: PCMA
Type: Audio
Clock rate: 80000 Hz
```
would be nicer?

Reply:
sure it's prettier, changed